packages/x-components/src/x-modules/search/wiring.ts

```diff
@@ -162,5 +176,11 @@ export const searchWiring = createWiring({
   },
   UserReachedResultsListEnd: {
     increasePageAppendingResults
+  },
+  ExtraParamsChanged: {
```

Comment:
Do we have to reset the rest of entities within the module? results, partialResults, relatedTags, banners, promoteds, selectedFilters, spellcheckedQuery ?

Reply:
I think as the request is send again this entities will be overridden right?

Reply:
As we've talked through Slack, you're right.